examples/stable-diffusion/image_to_video_generation.py

```diff
@@ -194,14 +213,20 @@ def main():
     )
     logger.setLevel(logging.INFO)
 
+    i2v_models = ["i2vgen-xl"]
+    i2v_model = True if any(model in args.model_name_or_path for model in i2v_models) else False
```

Comment:
```suggestion
    i2v_model = any(model in args.model_name_or_path for model in i2v_models)
```

Reply:
also it makes more sense to name it `is_i2v_model`

Reply:
Done.